resources/views/home.blade.php

```diff
@@ -2,19 +2,24 @@
     @guest
         @include('content.welcome')
     @endguest
-    <x-news.carousel/>
-    <?php
-    renderFinishedClaimsComponent(5);
-    RenderActivePlayersComponent();
-    ?>
-    <x-user.online-count-chart/>
-    <?php
-    renderNewClaimsComponent(5);
-    RenderRecentForumPostsComponent();
-    ?>
+
+    <x-news.carousel />
+    <x-claims.finished-claims count="6" />
+
+    @php
```

Comment:
Personally I prefer sticking with the `<?php ?>` tag because the `@php` blade directive causes problems in some IDEs with formatting.
Can stay though imo.

Reply:
Understood, for consistency I have reverted these back to `<?php ?>`. I was not aware of the formatting issues caused by the blade directive. This has been changed in https://github.com/RetroAchievements/RAWeb/pull/1425/commits/8cd548615949816bd06e748a3a3faaa2ed0b8f3e.